src/ray/core_worker/core_worker.h

```diff
@@ -26,22 +26,18 @@
 /// The set of gRPC handlers and their associated level of concurrency. If you want to
 /// add a new call to the worker gRPC server, do the following:
 /// 1) Add the rpc to the CoreWorkerService in core_worker.proto, e.g., "ExampleCall"
-/// 2) Add a new handler to the macro below: "RAY_CORE_WORKER_RPC_HANDLER(ExampleCall, 1)"
-/// 3) Add a method to the CoreWorker class below: "CoreWorker::HandleExampleCall"
-#define RAY_CORE_WORKER_RPC_HANDLERS                               \
-  RAY_CORE_WORKER_RPC_HANDLER(AssignTask, 5)                       \
-  RAY_CORE_WORKER_RPC_HANDLER(PushTask, 9999)                      \
-  RAY_CORE_WORKER_RPC_HANDLER(DirectActorCallArgWaitComplete, 100) \
-  RAY_CORE_WORKER_RPC_HANDLER(GetObjectStatus, 9999)               \
-  RAY_CORE_WORKER_RPC_HANDLER(KillActor, 9999)                     \
-  RAY_CORE_WORKER_RPC_HANDLER(GetCoreWorkerStats, 100)
+/// 2) Add a new handler declaration to `CoreWorkerServiceHandler`
+///    in core_worker_server.h, e.g. "HandleExampleCall"
+/// 3) Add a new macro to RAY_CORE_WORKER_RPC_HANDLERS in core_worker_server.h, e.g.
```

Comment:
Can we collapse (2) and (3)? Seems like the declarations in `CoreWorkerServiceHandler` are completely boilerplate, so we could have `RAY_CORE_WORKER_RPC_HANDLERS` expand into them?

Reply:
I added another macro to to make it easier to add a new handler function declaration. 

Previously we don't have `CoreWorkerServiceHandler`, but it's just because the RPC service code directly references back to `CoreWorker`, which creates a cyclic dependency between `CoreWorkerGrpcService` and `CoreWorker`. This PR breaks that cyclic dependency by adding a `CoreWorkerServiceHandler` interface, this is similar other grpc service implementations, and also `CoreWorkerClient` has a `CoreWorkerClientInterface`, so I think it's probably OK to add it. And with the new added macro, it's very simple to add a new declaration.